### PR TITLE
Chore/misc updates 07 2024

### PR DESCRIPTION
**Pull request by jul-dan**

### CODE REVIEW

File: website/docs/using-qovery/deployment/running-and-deployment-statuses.md [hunk 1]
```diff
@@ -78,6 +78,15 @@ Within this page you will have a view of:
 
 By clicking on `Logs`, you will be redirected to the service logs specifically filtered for this pod.
 
+### Clear old job executions
+
+If you have old cronjobs or lifecycle jobs execution in error, your global job status will be in `Warning`.
```
Comment (acarranoqovery) [inline website/docs/using-qovery/deployment/running-and-deployment-statuses.md:83]: we only keep the last lifecycle job execution, in theory this should be available only for cronjob

File: website/static/img/deployment/clear_job_status.png
Comment (acarranoqovery) [inline website/static/img/deployment/clear_job_status.png:1]: the screenshot seems to be really stretched. but maybe it's github 🤔

**Review by acarranoqovery — APPROVED** [review]

File: website/static/img/deployment/clear_job_status.png
Comment (jul-dan) [reply website/static/img/deployment/clear_job_status.png:1]: image

Comment (jul-dan) [reply website/static/img/deployment/clear_job_status.png:1]: Displayed like this on my screen

File: website/docs/using-qovery/deployment/running-and-deployment-statuses.md (hunk 1 above)
Comment (jul-dan) [reply website/docs/using-qovery/deployment/running-and-deployment-statuses.md:83]: You can clean lifecycle job status too

**Pull request merged**